src-java/kilda-model/src/main/java/org/openkilda/model/HaFlow.java

```diff
@@ -396,11 +435,11 @@ public interface HaFlowData {
 
         void addPaths(HaFlowPath... paths);
 
-        Optional<HaSubFlow> getSubFlow(String subFlowId);
+        Optional<HaSubFlow> getHaSubFlow(String subFlowId);
 
-        Set<HaSubFlow> getSubFlows();
+        List<HaSubFlow> getHaSubFlows();
```

Comment:
1. here maybe you can return Collection to be more open
2. I prefer getSubflows